boost-maven/boost-boms/boost-mp20-apis-bom/pom.xml

```diff
@@ -0,0 +1,119 @@
+<!-- Copyright (c) 2018 IBM Corporation and others. All rights reserved. 
+	This program and the accompanying materials are made available under the 
+	terms of the Eclipse Public License v1.0 which accompanies this distribution, 
+	and is available at http://www.eclipse.org/legal/epl-v10.html Contributors: 
+	IBM Corporation - initial API and implementation -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+    <groupId>io.openliberty.boost</groupId>
+    <artifactId>boost-mp20-apis-bom</artifactId>
+    <!--  Versioned separately from the main project -->
+    <version>0.1-SNAPSHOT</version>
+    <packaging>pom</packaging>
+
+	<description>Booster MP20 APIs Bom</description>
+
+    <properties>
+        <!--  Java EE specs  -->
+        <cdi-version>2.0</cdi-version>
+        <jaxrs-version>2.1</jaxrs-version>
+        <jsonp-version>1.1</jsonp-version>
+        <jsonb-version>1.0</jsonb-version>
+        <annotation-version>1.2</annotation-version>
+
+        <!-- MicroProfile specs  -->
+        <config-version>1.3</config-version>
+        <ft-version>1.1</ft-version>
+        <health-version>1.0</health-version>
+        <metrics-version>1.1</metrics-version>
+        <jwt-version>1.1</jwt-version>
+        <openapi-version>1.0</openapi-version>
+        <rest-client-version>1.1</rest-client-version>
+        <opentracing-version>1.1</opentracing-version>
+
+    </properties>
+
+	<dependencyManagement>
+		<dependencies>
+	        <dependency>
+	            <groupId>javax.enterprise</groupId>
+	            <artifactId>cdi-api</artifactId>
+	            <version>${cdi-version}</version>
+	        </dependency>
+	        <dependency>
+	            <groupId>javax.ws.rs</groupId>
+	            <artifactId>javax.ws.rs-api</artifactId>
+	            <version>${jaxrs-version}</version>
+	        </dependency>
+	        <dependency>
+	            <groupId>javax.json</groupId>
+	            <artifactId>javax.json-api</artifactId>
+	            <version>${jsonp-version}</version>
+	        </dependency>
+	        <dependency>
+	            <groupId>javax.annotation</groupId>
+	            <artifactId>javax.annotation-api</artifactId>
+	            <version>${annotation-version}</version>
+	        </dependency>
```

Comment:
Unlike, say, cdi-api, this dependency is not used by one of our boosters.    I wonder if we've thought through the pros/cons of bringing it into the BOM, then?